src/ralph/accounts/views.py

```diff
@@ -76,30 +74,33 @@ def post(self, request, *args, **kwargs):
         if settings.INVENTORY_TAG_APPEND_DATE:
             date_tag = settings.INVENTORY_TAG + '_' + date.today().isoformat()
 
-        asset.tags.add(
-            settings.INVENTORY_TAG,
-            settings.INVENTORY_TAG_USER,
-            date_tag
-        )
-        asset.save()
+        if request.POST['answer'] == 'yes':
```

Comment:
IMO you should split this two block of code (inside ``if``s) to two separeted methods like ``_post_yes(self, request, asset)`` and ``_post_no(self, request, asset)``. Benefits? Easliy to maintence, testing, debugging and read.